resources/views/entities/pages/posts/_form.blade.php

```diff
@@ -45,6 +49,9 @@
                 @can('permission', $entity->child)
                     <x-tab.tab target="permissions" :title="__('entities/notes.show.advanced')"></x-tab.tab>
                 @endcan
+                @if(auth()->user()->isAdmin() && !empty($templates))
```

Comment:
Please make this a new policy on the Campaign called 'useTemplates' that returns the isAdmin() check for now. Also adapt the templates dropdown on the crud lists of entities to use that (there is a ticket for allowing non-admins to set and use templates that we will work on later this month)